Makefile

```diff
@@ -61,3 +62,14 @@ pdk-phase-checks:
 	luacov -c t/phase_checks.luacov
 	grep "ngx\\." t/phase_checks.report
 	grep "check_" t/phase_checks.report
+
+win_scripts:
+	@for script in $(WIN_SCRIPTS) ; do \
+	  if [ $$(grep -obUaPc '\015' $$script) -gt 0 ] ; then \
```

Comment:
grep is notoriously inconsistent across platforms; the "alphabet soup" of those flags is dangerous (especially if is a dependency of all build targets above).

In particular, I just tested it locally and `-P` doesn't work on my machine. Other distros may have other surprises.

How about making this `win_scripts` unconditional (ie, removing the `if [ $$(grep...` line) and not running it as a dependency of every target (but keeping it in the makefile so it's available there for a quick fixup if needed by Windows users)?

Reply:
That's actually a good idea, I'll update it